External/Plugins/CodeRefactor/Commands/RenameFile.cs

```diff
@@ -44,19 +44,20 @@ protected override void ExecutionImplementation()
                     return;
                 }
             }
+
+            string filePath = oldPath;
             // refactor failed or was refused
             if (Path.GetFileName(oldPath).Equals(newPath, StringComparison.OrdinalIgnoreCase))
             {
                 // name casing changed
-                string tmpPath = oldPath + "$renaming$";
-                File.Move(oldPath, tmpPath);
-                oldPath = tmpPath;
+                filePath += "$renaming$";
+                File.Move(oldPath, filePath);
             }
             if (!Path.IsPathRooted(newPath)) newPath = Path.Combine(Path.GetDirectoryName(oldPath), newPath);
 
             if (FileHelper.ConfirmOverwrite(newPath))
             {
-                FileHelper.ForceMove(oldPath, newPath);
+                FileHelper.ForceMove(filePath, newPath);
                 DocumentManager.MoveDocuments(oldPath, newPath);
```

Comment:
The issue here was that `oldPath` contained `$renaming$`, so document manager couldn't find any matching documents, and the commit message generated by the source control dialog contained this unwanted `$renaming$`